recipes/python-cobra/bld.bat

```diff
@@ -0,0 +1,2 @@
+"%PYTHON%" setup.py install --single-version-externally-managed --record record.txt
+if errorlevel 1 exit 1
```

Comment:
We can remove this as `conda-build` now respects the `script` key in `meta.yaml` for Windows builds.